src/patterns/dosdp-dev/abnormalConstrictionOfAnatomicalEntity.yaml

```
@@ -38,6 +45,6 @@ def:
   - anatomical_entity
 
 equivalentTo:
-  text: "'has_part' some ('constricted' and ('inheres_in' some %s) and ('has_modifier' some 'abnormal'))"
+  text: "'has_part' some ('constricted' and ('characteristic_of_part of' some %s) and ('has_modifier' some 'abnormal'))"
```

```suggestion
  text: "'has_part' some ('constricted' and ('characteristic_of_part_of' some %s) and ('has_modifier' some 'abnormal'))"
```
There is a missing `_` here. I think this is why it fails QC. You can validate your dos-dp pattern template on your laptop like this:
```
dosdp validate -i abnormalConstrictionOfAnatomicalEntity.yaml
```